platform/mbed_preprocessor.h

```diff
@@ -52,6 +52,33 @@
 #define MBED_STRINGIFY(a) MBED_STRINGIFY_(a)
 #define MBED_STRINGIFY_(a) #a
 
+/** MBED_STRLEN
+ *  Reports string token length
+ *
+ *  @note
+ *  Expands tokens before calculating length
+ *
+ *  @code
+ *  // Get string length
+ *  const int len = MBED_STRLEN("Get the length")
+ *  @endcode
+ */
+#define MBED_STRLEN(a) MBED_STRLEN_(a)
+#define MBED_STRLEN_(a) (sizeof(a) - 1)
+
+/** MBED_COUNT_VA_ARGS(...)
+ *  Reports number of tokens passed
+ *
+ *  @note
+ *  Token limit is 16
```

Comment:
What's the reason for this limit?

Reply:
No particular reason for 16, we can have higher but will need some limit for macro to work

Reply:
In current state, if arguments more then 16 are passed response will be 16 the max limit